lib/frisby.js

```diff
@@ -725,21 +725,21 @@ Frisby.prototype.expectJSON = function(jsonTest) {
   return this;
 };
 
-
 // Helper to validate JSON response against provided JSONSchema structure or document
-Frisby.prototype.expectJSONSchema = function(path, jsonSchema) {
+Frisby.prototype.expectJSONSchema = function(path, jsonSchema, context) {
   var self       = this,
       args       = Array.prototype.slice.call(arguments),
-      path       = typeof args[0] === 'string' && args.shift(),
-      jsonSchema = (typeof args[0] === 'object' || typeof args[0] === 'string') && args.shift();
```

Comment:
True, but can you come up with ways that do not break it? Also, having arguments that can be whatever is not a very sane way of using function overloading in javascript while it is not supported? imho, arguments should not be a catchall and the function should not "guess" what they are. I got into a dilemma as I added a new argument as the amount of combinations possible become very big.